src/app/app.module.ts

```diff
@@ -35,7 +35,13 @@ import { WirelessDetailComponent } from './wireless/wireless-detail/wireless-det
 import { SolModule } from '@open-amt-cloud-toolkit/ui-toolkit-angular/sol'
 import { KvmModule } from '@open-amt-cloud-toolkit/ui-toolkit-angular/kvm'
 import { DeviceUserConsentComponent } from './devices/device-user-consent/device-user-consent.component'
-
+import { EventChannelComponent } from './devices/event-channel/event-channel.component'
+import { IMqttServiceOptions, MqttModule } from 'ngx-mqtt'
+export const MQTT_SERVICE_OPTIONS: IMqttServiceOptions = {
+  hostname: 'localhost',
```

Comment:
lets pull this from the environment.ts file

Reply:
done